autogpt_platform/backend/backend/data/graph_test.py

```
@@ -2373,3 +2373,210 @@ async def test_get_graph_without_org_keeps_strict_ownership(mocker):
+def test_only_known_blocks_use_discriminator_type_mapping():
+    """Guard on the fallback's trigger condition. If a new block adds
+    discriminator_type_mapping, it silently inherits schema-default
+    discrimination — this test forces that to be a conscious decision."""
+    from backend.blocks import load_all_blocks
+
+    users = set()
+    for block_cls in load_all_blocks().values():
+        try:
+            block = block_cls()
+        except Exception:
+            continue
```

Ruff (0.16.1)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)